smtlib-backends-process/src/SMTLIB/Backends/Process.hs

```diff
@@ -63,16 +65,16 @@ new ::
   Config ->
   IO Handle
 new Config {..} = decorateIOError "creating the solver process" $ do
-  (Just hIn, Just hOut, Just hErr, process) <-
+  (Just hIn, Just hOut, hMaybeErr, process) <-
     P.createProcess
       (P.proc exe args)
-        { std_in = P.CreatePipe,
-          std_out = P.CreatePipe,
-          std_err = P.CreatePipe
+        { P.std_in = P.CreatePipe,
+          P.std_out = P.CreatePipe,
+          P.std_err = std_err
         }
-  mapM_ setupHandle [hIn, hOut, hErr]
+  mapM_ setupHandle [hIn, hOut]
```

Comment:
```suggestion
  mapM_ setupHandle [hIn, hOut]
  maybe (return ()) setupHandle hMaybeErr
```

It's not strictly necessary but I don't think it hurts to add this.

Reply:
Actually nevermind, we probably want to stick with text mode for the error channel.